src/containers/Sidebar/index.tsx

```diff
@@ -49,7 +50,13 @@ export class SidebarPresentation extends React.Component<Props> {
     this.props.setActiveAccount({ targetAccount: account, accounts: this.props.accounts })
   }
 
-  public addAccount = () => this.props.addNextAccountFromSeedphrase()
+  public addAccount = () => {
+    this.props.addNextAccountFromSeedphrase()
+    sendAnalyticsEvent({
```

Comment:
Might be worth checking with Sam B if we want to call this event regardless of whether it succeeds or fails? Currently it fires even if the addAccount fails. My preference would be to call this from inside the epic as well, for two reasons: we are already calling it there for the transfer success/failure, but also because we can then call it only when the add account succeeds. Potentially also we could call it with a failure type in the epic easily, if Sam would like that.